Triphouse-project/style.css

```diff
@@ -416,76 +429,214 @@ input:focus {
     .services-mobile {
         display: block;
     }
-    
+
     .services {
         display: none;
-    }
+    }    
     
     .container-mobile {
         padding: 24px 73px 40px;
     } 
 
-    .services__title {
+    .services-mobile__title {
         text-align: center;
     }
-
-    .services__title h2 {
+    
+    .services-mobile h2 {
         font-size: 26px;
         font-style: normal;
         font-weight: 500;
     }
-    
-    
-    .services__list {
+
+    .services-mobile__list {
         display: flex;
         flex-direction:column;
         margin-top: 36px;
         align-items: center;
     }
 
-    .support {
+    .services-mobile__item .support {
         background-image:url(./css_sprites-mobile.png);
         background-position: -10px -90px;
         width: 54px;
         height: 66px;
         margin: 0 150px 19px;
     }
 
-    .comunication {
+    .services-mobile__item .comunication {
         background-image: url(./css_sprites-mobile.png);
         background-position: -96px -10px;
         width: 60px;
         height: 60px;
         margin: 37px 150px 22px;
     }
-    
-    .booking {
+
+    .services-mobile__item .booking {
         background-image: url(./css_sprites-mobile.png);
         background-position: -84px -90px;
         width: 70px;
         height: 48px;
         margin: 44px 150px 28px;
     }
     
-    .reviews {
+    .services-mobile__item .reviews {
         background-image: url(./css_sprites-mobile.png);
         background-position: -10px -10px;
         width: 66px;
         height: 60px;
         margin: 35px 150px 25px;
     }
 
-    .services__item p {
+    .services-mobile__item p {
         font-size: 14px;
         font-style: normal;
         font-weight: 400;
         text-align: center;
-        margin-top: 20px;
+        margin-top: 25px;
     }
-    
-    
 }
 
+.recommendations {
+    background-color: #EAF0F9;
+}
+
+.recommendations__title {
+    text-align: center;
+    padding-top: 120px;
+}
+
+.recommendations h2 {
+    font-size: 40px;
+    font-style: normal;
+    font-weight: 500;
+    color: #383838;
+}
+
+.recommendations__list {
+    display: flex;
+    margin-top: 106px;
+    justify-content: space-between;
+    padding-bottom: 120px;
+    position: relative;
+}
+
+.recommendations__list li {
+    width: 23.5%;
+}
+
+.recommendations__list li img {
+    margin-bottom: 24px;
```

Comment:
![2021-04-26_18h21_15](https://user-images.githubusercontent.com/19935417/116108286-7318ce00-a6bc-11eb-8d64-48e906aaf198.png)
Add for prettier image size

Reply:
fixed